environment.devenv.yml

```diff
@@ -0,0 +1,10 @@
+name: oop-ext
+
+dependencies:
+  - python>=3.6
+  - pytest
+  - pytest-mock>=1.10
+
+environment:
+  PYTHONPATH:
+    - root + '/src'
```

Comment:
```suggestion
    - {{ root }}/src
```

Reply:
Check Yaml...............................................................Failed
hookid: check-yaml

while parsing a block collection
  in "environment.devenv.yml", line 10, column 5
expected <block end>, but found '<scalar>'
  in "environment.devenv.yml", line 10, column 17

Reply:
Sorry what?

Reply:
Removed check-yaml hook from pre-commit